<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>century gothic</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
                font-size:20px;
                font-family:century gothic;
                }
                QFrame{
                background:#333;
                }
                #centralwidget{
                border-image: url(sources/pic/snowhouse.jpg) no-repeat;
                }
                QPushButton{
                background:#CC3333;
                border-radius:20px;
                color:white;
                }

                QToolButton{
                background:red;
                border-radius:20px;
                color:white;
                }

                QPushButton:hover{
                background:#ff6666;
                }
                #title{
                color:white;
                border-bottom:none;
                }
                QCheckBox{
                background:transparent;
                color:white;
                }
                QLineEdit{
                background:transparent;
                }
                QLabel{
                border:none;
                color:#717072;
                border-bottom:1px solid #717072;
                }
                #titleButton:hover{
                background:#ff6666
                }
QComboBox{
color:red;
background-color: #666;
	border:1px solid rgb(221,221,221);
	font: 10pt &quot;微软雅黑&quot;;
	padding-left:10px;
}
QComboBox::drop-down{
background-color: rgb(0, 0, 0);
}
     
  </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>190</y>
      <width>500</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="anlzButton">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>480</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>解析Excel</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>30</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel生成器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>430</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>加载Excel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pathLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:12px</string>
     </property>
     <property name="text">
      <string>文件地址:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="conditionBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>159</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>当</string>
     </property>
    </widget>
    <widget class="QComboBox" name="operatorBox">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>159</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="valueEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>152</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;
border-color:red</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>151</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>时:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>53</x>
       <y>210</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#666;
color:#FFF;
border:2px;
border-color:red</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="titleButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TIPS</string>
    </property>
    <property name="iconSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="QySide2/side.qrc"/>
 </resources>
 <connections/>
</ui>
